tools/fixup_stuff.sh

```diff
@@ -202,5 +202,9 @@ fi
 # on python-virtualenv), first install the distro python-virtualenv
 # to satisfy any dependencies then use pip to overwrite it.
 
-install_package python-virtualenv
+if is_clearlinux; then
```

Comment:
Just change this to if ! is_clearlinux;, no need for the echo here

Reply:
Done